src/cluster.c

```diff
@@ -264,6 +377,8 @@ int clusterLoadConfig(char *filename) {
         /* The plaintext port for client in a TLS cluster (n->pport) is not
          * stored in nodes.conf. It is received later over the bus protocol. */
 
+        sdsfreesplitres(aux_argv, aux_argc);
+
```

Comment:
My understanding is that the plaintext port comes from the server config. If we persist it here again, then we will have to deal with any potential conflicts. Curios to know what value does it bring us if we were to persist it in nodes.conf as well?

Reply:
The pport is a server config so the server which has it configured will know it, but if other nodes persist it too, they are able to report it in e.g. CLUSTER SLOTS after a restart, before they get a gossip message from the node with pport configured.

Reply:
This sounds like a useful thing if nodes in the same cluster are configured with different ports. Is this also related to the deployment that uses NAT or port forwarding?

Reply:
I would also like to see us store the plaintext port. I think I mentioned it in the original port meta data PR that on restart we will not show the correct port for a bit. It would be a good improvement.

Reply:
Sounds good to me. Logged #11260 to track the proposal.